src/dparse/ast.d

```diff
@@ -75,6 +75,9 @@ shared static this()
     typeMap[typeid(XorExpression)] = 48;
 }
 
+/// Statement block style
+enum StatementBlockStyle : ubyte { single, colon, block }
```

Comment:
StatementBlockStyle wasn't a good name idea by me because there is already BlockStatement and other statements are single, so this should rather be `DeclarationListStyle` and also document its members because this is how it is used.
```suggestion
enum DeclarationListStyle : ubyte
{
	/// A declaration directly after the containing AST node making it the only child
	single,
	/// A colon (`:`) was used in the containing AST node meaning all following declarations are part here.
	colon,
	/// The declarations have been specified in a block denoted by starting `{` and ending `}` tokens.
	block
}
```